web/src/invite.ts

```diff
@@ -22,6 +22,7 @@ import * as email_pill from "./email_pill";
 import {$t, $t_html} from "./i18n";
 import * as input_pill from "./input_pill";
 import * as invite_stream_picker_pill from "./invite_stream_picker_pill";
+import {add_default_stream_pills} from "./invite_stream_picker_pill";
```

Comment:
No need for the extra import here—you can just reference it directly as `invite_stream_picker_pill.add_default_stream_pills`.